### PR TITLE
Kl 11 game logic

### PR DESCRIPTION
**Pull request by tulza**
closes #12
a lot of stuff XD, ur problem now
90% working

### CODE REVIEW

**Comment by tulza** [comment]
ps, add timer to count up for me for characters mode then run Gamedata.finishGame();

**Comment by AntGa** [comment]
crazy work 80 files changes

File: .env-example
Comment (AntGa) [inline .env-example:1]: nice

File: src/components/common/ui/game/GameBoard.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import React, { MouseEvent, useRef } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
```
Comment (AntGa) [inline src/components/common/ui/game/GameBoard.tsx:4]: crazy work

File: src/app/login/SignUpForm.tsx [hunk 2]
```diff
@@ -27,16 +27,16 @@ const SignUpForm = () => {
...
+  });
 
   // Handle form submission
   async function onSubmit(values: z.infer<typeof signUpSchema>) {
-    const res = await signUp(values)
+    const res = await signUp(values);
     if (res.success) {
-      toast.success('Account created successfully')
-      router.push('/dashboard')
+      toast.success('Account created successfully');
```
Comment (AntGa) [inline src/app/login/SignUpForm.tsx:36]: getting baited by prettier formatting :sob:

**Review by AntGa — APPROVED** [review]
lgtm

**Pull request merged**